packages/cli/src/commands/generate/page/templates/stories.tsx.parametersTemplate

```diff
@@ -0,0 +1,12 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react'
+
+import ${pascalName}Page from './${pascalName}Page'
+
+export const generated: ComponentStory<typeof ${pascalName}Page> = (args) => {
+  return <${pascalName}Page ${propValueParam} {...args} />
```

Comment:
Am I correct in thinking that `propValueParam` is used to provide a default value, but you can then let `{...args}` spread over it to render with a different value for that prop?

Reply:
Yeah that's right!

https://user-images.githubusercontent.com/32992335/186808642-b3749999-60ed-4663-b65d-430441246033.mov